go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/appuio/seiso
 go 1.14
 
 require (
+	github.com/aws/aws-sdk-go v1.30.9
```

Comment:
Where's this coming from?

Reply:
Interesting, haven't payed attention to that, I deleted the dependency and it seems to work